keyboards/naked48/rev1/rev1.h

```diff
@@ -0,0 +1,91 @@
+#ifndef REV1_H
+#define REV1_CONFIG_H
+
+#include "../naked48.h"
```

Comment:
The main keyboard file is added to the VPATH, so you don't need to specify the path, just the file name.
```suggestion
#include "naked48.h"
```